internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -635,27 +668,36 @@ func NewOpenShiftAuthProxyContainer(cr *model.CryostatInstance, specs *ServiceSp
 		fmt.Sprintf("--upstream=http://localhost:%d/", constants.CryostatHTTPContainerPort),
 		fmt.Sprintf("--upstream=http://localhost:%d/grafana/", constants.GrafanaContainerPort),
 		fmt.Sprintf("--upstream=http://localhost:%d/storage/", constants.StoragePort),
-		"--cookie-secret=REPLACEME",
 		fmt.Sprintf("--openshift-service-account=%s", cr.Name),
 		"--proxy-websockets=true",
 		fmt.Sprintf("--http-address=0.0.0.0:%d", constants.AuthProxyHttpContainerPort),
-		fmt.Sprintf(
-			`--openshift-sar=[{"group":"","name":"","namespace":"%s","resource":"pods","subresource":"exec","verb":"create","version":""}]`,
-			cr.InstallNamespace,
-		),
-		fmt.Sprintf(
-			`--openshift-delegate-urls={"/":{"group":"","name":"","namespace":"%s","resource":"pods","subresource":"exec","verb":"create","version":""}}`,
-			cr.InstallNamespace,
-		),
-		"--bypass-auth-for=^/health",
 		"--proxy-prefix=/oauth2",
 	}
+	if isOpenShiftAuthProxyDisabled(cr) {
+		args = append(args, "--bypass-auth-for=.*")
+	} else {
+		args = append(args, "--bypass-auth-for=^/health")
```

Comment:
Do we need something like `$` at the end of the regex here? We probably won't get into a situation where there's a valid path like `/healthfoo` that's not supposed to bypassed, but it might be good just in case.